src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,125 @@
+                String date = splitDate[1];
+                String[] breakingDate = date.split("/");
+                String year = breakingDate[2];
+                String month = breakingDate[1];
+                String currentDate = breakingDate[0];
+                int i = Integer.parseInt(currentDate);
+                if (i < 10) {
+                    currentDate = "0" + currentDate;
+                }
+                String finalDateFormat = year + "-" + month + "-" + currentDate;
+                LocalDate date1 = LocalDate.parse(finalDateFormat);
+                String dateForObject = date1.format(DateTimeFormatter.ofPattern("MMM d yyyy"));
+                t.addDeadline(firstDeadline, date1);
+                System.out.println("Got it. I've added this task: ");
+                System.out.println(new Deadline(firstDeadline, date1));
+                System.out.println("Now you have " + t.size() + " tasks in the list");
+                s.appendListToFile(t);
+            }
+        } else if (input.startsWith("event")) {
+            if (input.length() < 7) {
+                System.out.println(new NullTaskError().getMsg("event"));
+            } else {
+                String[] tempEvent = input.split("/at");
+                String firstEvent = tempEvent[0].substring(6);
+                System.out.println(firstEvent);
+                String[] splitDate = tempEvent[1].split(" ");
+                String date = splitDate[1];
+                String[] breakingDate = date.split("/");
+                String year = breakingDate[2];
+                String month = breakingDate[1];
+                String currentDate = breakingDate[0];
+                int i = Integer.parseInt(currentDate);
+                if (i < 10) {
+                    currentDate = "0" + currentDate;
+                }
+                String finalDateFormat = year + "-" + month + "-" + currentDate;
+                LocalDate date1 = LocalDate.parse(finalDateFormat);
+                System.out.println(date1);
+                t.addEvent(firstEvent, date1);
+                System.out.println("Got it. I've added this task: ");
+                System.out.println(new Event(firstEvent, date1));
+                System.out.println("Now you have " + t.size() + " tasks in the list");
+                s.appendListToFile(t);
+            }
+        } else if (input.startsWith("delete") && input.length() < 11) {
+            int value = Integer.parseInt(input.replaceAll("[^0-9]", ""));
+            Task removedTask = t.get(value - 1);
+            t.delete(value - 1);
+            System.out.println("Noted. I've removed this task:");
+            System.out.println(removedTask.toString());
+            System.out.println("Now you have " + t.size() + " tasks in the list.");
+            s.appendListToFile(t);
+        } else if (input.startsWith("find")) {
+            String keyword = input.substring(5);
+            ArrayList<Task> output = new ArrayList<>();
```

Comment:
Perhaps the usage of plural form for output will work better?

Reply:
I think the variable name should be completely renamed. The queried tasks aren't an "output"; they are the result of a query, and are then reformatted into a text result that gets printed.